Makefile

```
@@ -0,0 +1,31 @@
+-include .env.local
+.EXPORT_ALL_VARIABLES:
+MAKEFLAGS += --no-print-directory
+
```

```suggestion
-include .env.local
.EXPORT_ALL_VARIABLES:
MAKEFLAGS += --no-print-directory

install:
        yarn
	foundryup
	git submodule update --init --recursive
```
So that calling `make` from a fresh repository allows you to call other forge commands. 

Actually this is the only target that I would be adding to the makefile. As @MathisGD, this is not a blocker for me if we keep the other targets